modules/tinymce/src/plugins/table/main/ts/queries/TableTargets.ts

```diff
@@ -5,53 +5,44 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Element as DomElement, Node } from '@ephox/dom-globals';
 import { Fun, Option } from '@ephox/katamari';
+import { Element } from '@ephox/sugar';
 import * as CellOperations from './CellOperations';
+import { RunOperation } from '@ephox/snooker';
 
-const noMenu = function (cell) {
+const noMenu = function (cell: Element<DomElement>): RunOperation.CombinedTargets {
   return {
     element: Fun.constant(cell),
     mergable: Option.none,
     unmergable: Option.none,
     selection: Fun.constant([ cell ])
   };
-};
+}; 
```

Comment:
Trailing blank space that shouldn't be here (this is one of the reasons eslint is failing).